utils/generate_types/index.js

```diff
@@ -211,10 +211,19 @@ function createEventDescriptions(classDesc) {
 function classBody(classDesc) {
   const parts = [];
   const eventDescriptions = createEventDescriptions(classDesc);
+  const commentForMethod = {
+    on: 'Adds an event listener.',
+    addListener: 'Adds an event listener.',
+    off: 'Removes an event listener.',
+    removeListener: 'Removes an event listener.',
+    once: 'Adds an event listener that will be automatically removed after it is triggered once.'
+  }
   for (const method of ['on', 'once', 'addListener', 'removeListener', 'off']) {
     for (const {eventName, params, comment} of eventDescriptions) {
-        if (comment)
+        if ((method === 'on' || method === 'addListener') && comment)
```

Comment:
`once` might make sense there too, or do you think its more rarely used?

Reply:
I think users are more like to know what the event they are listening to does than know what once does.